cmd/scollector/collectors/sql_windows.go

```diff
@@ -284,3 +301,113 @@ type Win32_PerfRawData_MSSQLSERVER_SQLServerDatabases struct {
 	TransactionsPersec            uint64
 	WriteTransactionsPersec       uint64
 }
+
+func c_mssql_replica_db() (opentsdb.MultiDataPoint, error) {
+	var dst []Win32_PerfRawData_MSSQLSERVER_SQLServerDatabaseReplica
+	var q = wmi.CreateQuery(&dst, "WHERE Name <> '_Total'")
+	err := queryWmi(q, &dst)
```

Comment:
Join this line with the next since you don't need the return values after the if block:

if err := q(...); err != nil { ... }